src/compiler/glue-compiler.cc

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) 2020-2021 by the Zeek Project. See LICENSE for details.
 
-#include "glue-compiler.h"
+#include <zeek-spicy/compiler/glue-compiler.h>
```

Comment:
Let's use an include in `"` for the matching header. This is e.g., used by clang-format or clangd to detect matching header/source pairs.